pysat/tests/test_meta.py

```diff
@@ -2045,3 +2055,40 @@ def test_transfer_attributes_to_instrument_leading_underscore(self):
         assert self.meta._hidden_attribute == 'is it me'
         assert self.meta.__private_attribute == "you're looking for"
         return
+
+
+class TestDeprecation(object):
+    """Unit tests for DeprecationWarning."""
+
+    def setup(self):
+        """Set up the unit test environment for each method."""
+
+        warnings.simplefilter("always", DeprecationWarning)
+        self.meta = pysat.Meta()
```

Comment:
Recommend adding all necessary class attributes here.

Reply:
we don't need them for this test, but they could be added in the name of future expansion.  Or I could add a comment about generating an empty meta object.